src/platform_impl/windows/drop_handler.rs

```diff
@@ -108,19 +107,19 @@ impl FileDropHandler {
         };
         *pdwEffect = drop_handler.cursor_effect;
 
-        S_OK
+        0
```

Comment:
Use `windows_sys::Win32::Foundation::S_OK` (this goes for all the places where `S_OK` has been replaced)